javascript/javascript3/week2/promiseResolvesAfterTime.js

```diff
@@ -0,0 +1,25 @@
+let resolvePromiseAfter; //global variable for promise
```

Comment:
How can we avoid a global variable here?
We always try to enapsulate everything in programming